de.fhdo.lemma.service.open_api/plugin.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure if Eclipse requires a (possibly empty) plugin.xml file for plugins to work. Could you check and remove this empty plugin.xml if it is not needed at all?